src/middlewared/middlewared/plugins/disk_/info.py

```diff
@@ -43,16 +43,8 @@ async def get_partition_uuid_from_name(self, part_type_name):
 
     @private
     async def get_valid_zfs_partition_type_uuids(self):
-        # https://salsa.debian.org/debian/gdisk/blob/master/parttypes.cc for valid zfs types
-        # 516e7cba was being used by freebsd and 6a898cc3 is being used by linux
-        return [
-            '6a898cc3-1dd2-11b2-99a6-080020736631',
-            '516e7cba-6ecf-11d6-8ff8-00022d09712b',
-        ]
+        return ['516e7cba-6ecf-11d6-8ff8-00022d09712b']
```

Comment:
Is still a valid zfs partition, no? pool might have been created on Linux but used on BSD

Reply:
True, I fixed.